include/sqlpp11/mysql/connection.h

```diff
@@ -35,6 +35,7 @@
 #include <sqlpp11/serialize.h>
 #include <sstream>
 #include <string>
+#include <mysql/mysql.h>
```

Comment:
I have tried to use <mysql.h>, but after make install, I try to test this function it will say
```
In file included from /usr/local/include/sqlpp11/mysql/mysql.h:30:0,
                 ...
/usr/local/include/sqlpp11/mysql/connection.h:38:10: fatal error: mysql.h: No such file or directory
 #include <mysql.h>
          ^~~~~~~~~
```
how should I fix this?

Reply:
Your original plan was to offer a function that would expose the the native handle (MYSQL*).
By forward-declaring MYSQL, you could avoid including the header here.